boshio/boshio.go

```diff
@@ -194,7 +196,10 @@ func (c Client) retryableRequest(stemcellURL string, byteRange string) ([]byte,
 	byteRangeHeader := fmt.Sprintf("bytes=%s", byteRange)
 	req.Header.Add("Range", byteRangeHeader)
 
-Retry:
+	return c.retry(req)
+}
+
+func (c Client) retry(req *http.Request) ([]byte, error) {
```

Comment:
Done, ended up looking about the same although it is now more obvious that it will retry an infinite number of times.